packages/util/src/Environment.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
rather than doing this, i think we could use the platform CLI flag.

eg. see how we do it here: https://github.com/proofcarryingdata/zupass/blob/9a3bbd25ee8f49f9ab805756e179639c12114a1a/packages/ethereum-group-pcd/package.json

Reply:
Oh interesting!

Reply:
> So, I think we should use the conditional exports anyway in all of our packages, but that doesn't handle the env condition in this PR's code. And we can solve it in two ways: checking the environment at run-time as I've done, or at build-time, like uuid does here (basically replacing files with browser/node code depending on the environment).

OK, checking it at runtime seems like the better option between the two. Is the way you did it following best-practices? I don't have context on what the best practices would be for this.